app/lib/submission_builder/ty2022/states/ny/individual_return.rb

```diff
@@ -68,6 +68,23 @@ def documents_wrapper
                 end
               end
 
+              @submission.data_source.dependents.where(eic_qualifying: true).each_with_index do |dependent, index|
+                xml.dependent do
+                  xml.DEP_SSN_NMBR dependent.ssn
+                  xml.DEP_SEQ_NMBR index+1
+                  xml.DEP_DISAB_IND dependent.eic_disability == true ? 1 : 2
+                  xml.DEP_FORM_ID 215
+                  xml.DEP_RELATION_DESC dependent.relationship
+                  xml.DEP_STUDENT_IND dependent.eic_student == true ? 1 : 2
+                  xml.DEP_CHLD_LAST_NAME dependent.last_name
+                  xml.DEP_CHLD_FRST_NAME dependent.first_name
+                  xml.DEP_CHLD_MI_NAME dependent.middle_initial
+                  xml.DEP_CHLD_SFX_NAME dependent.suffix
+                  xml.DEP_MNTH_LVD_NMBR dependent.months_in_home
+                  xml.DOB_DT dependent.dob&.strftime("%Y-%m-%d")
+                end
+              end
+
```

Comment:
this makes these dependent nodes for IT-215
![Screenshot 2023-11-06 at 3 13 53 PM](https://github.com/codeforamerica/vita-min/assets/49880002/b19bbc80-68db-4f6e-8515-127d9eb8bd00)